requirements.txt

```diff
@@ -0,0 +1,6 @@
+matplotlib
+numpy
+pandas
+pandas_datareader
+pyyaml
+tqdm
```

Comment:
requirements.txt is a kind of legacy way to manage scripts as package.
we have to consider to start using setup.py or some other way